packages/accordions/src/styled/StyledStep.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledContent } from './StyledContent';
+import { StyledLine } from './StyledLine';
+
+const COMPONENT_ID = 'accordions.step';
+
+export interface IStyledStep {
+  isHorizontal?: boolean;
+}
+
+export const StyledStep = styled.li.attrs<IStyledStep>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledStep>`
+  position: ${props => props.isHorizontal && 'relative'};
+  flex: ${props => props.isHorizontal && '1'};
+  min-width: ${props => props.isHorizontal && `${props.theme.space.base * 15}px`};
+
+  &:last-of-type ${StyledLine} {
+    display: ${props => props.theme.rtl && 'none'};
+  }
+
+  &:first-of-type ${StyledLine} {
+    display: ${props => !props.theme.rtl && 'none'};
+  }
+
+  &:not(:last-of-type) ${StyledContent} {
+    /* stylelint-disable-next-line property-no-unknown */
+    border-${props => (props.theme.rtl ? 'right' : 'left')}: ${props => props.theme.borders.sm};
+    border-color: ${props => getColor('neutralHue', 300, props.theme)};
+  }
```

Comment:
Now that we see how line wrapping impacts vertical layouts, I wonder if you'll need a variation on `StyledLine` instead of this border treatment? 😬 

![Screen Shot 2020-03-10 at 11 22 46 AM](https://user-images.githubusercontent.com/143773/76346355-1d1f7600-62c2-11ea-9bf2-a5a4b68a4574.png)

/cc @ginnywood

Reply:
Another option would is the vertical stepper could keep the label on a single line with a `min-width`. This would cause the label to overflow the `Stepper`. In the meantime, I can see about another variation on `StyledLine`, but that will be difficult since the line will need to be sized based on the amount of content.

Reply:
Yeah. Get design direction on this before you chase down any particular solution.

Reply:
> Another option would is the vertical stepper could keep the label on a single line with a `min-width`. This would cause the label to overflow the `Stepper`
Can I see what that would be like?

Reply:
@ginnywood an arbitrary `min-width` would look like this:

(`min-width: 600px` used):
<img width="975" alt="Screen Shot 2020-03-10 at 1 42 03 PM" src="https://user-images.githubusercontent.com/1811365/76357606-41388280-62d5-11ea-8f84-61f84d6b5ea2.png">

The label _would_ still wrap if the text overflows the arbitrary `min-width` (`600px` in the case of these screenshots). For example, here is a paragraph used as a label:

<img width="975" alt="Screen Shot 2020-03-10 at 1 43 16 PM" src="https://user-images.githubusercontent.com/1811365/76357801-9379a380-62d5-11ea-8e4a-29ff49ab23db.png">

Reply:
You would need to impose `white-space: nowrap` on these labels to prevent the wrapping.

Reply:
I’m concerned about not allowing wrapping. It seems like it will obscure information.

I would rather have the lines break at these extreme sizes than obscure information. However, before resorting to that, could a styling change to the line help here? From a design perspective, it is negotiable. We could remove the gap between the line and the icon...we could explore removing the line...whatever would be helpful.

Reply:
@ginnywood I think line-wrapping is the right call. Can we agree to punt the 1px border spacing decision to a follow-on? There are a frustrating number of structural implications that need to be considered in order to handle that.

You should feel free to make the most optimal design decision. I think it would be best if we can postpone development under a separate PR.

Reply:
Absolutely!

Reply:
Thanks for the discussion @ginnywood @jzempel! Let's reopen this thread when the follow-on occurs.